lib/DMVfacility.rb

```diff
@@ -0,0 +1,30 @@
+class DMVfacility
+    
+    def create_facility(datas)
+        datas.map do |data|
+            Facility.new({
+                name: name(data),
+                address: address(data), 
+                phone: phone(data)})
+        end
+    end
+
+    def name(database)
```

Comment:
Good use of these helper methods to help you sort through the api data. I'm not sure `database` is the best name for a variable here, maybe `data` or `data` would be better.